### ISSUE TITLE
Change RELAX NG reference

### ISSUE THREAD

**Issue by rubensworks** [w3c/sparql-results-xml#17]
As suggested by @domel in #13:

> Can we change a RELAX NG reference? Now it refers to OASIS spec. There is also an ISO standard (ISO/IEC 19757-2:2003). OASIS is open but old. ISO version is more official and newer, but it is also close access.

**Comment by gkellogg** [comment]
The place to do this would be in [Specref](https://github.com/tobie/specref), as it's defined there, and used at least in RDF/XML as well, if not many other places.

["RELAXNG"](https://github.com/tobie/specref/blob/ebad7d8f7fdd1250451d69f32f3c87d9507914f1/refs/biblio.json#L2531-L2540)

["RELAXNG-COMPACT"](https://github.com/tobie/specref/blob/ebad7d8f7fdd1250451d69f32f3c87d9507914f1/refs/biblio.json#L2541-L2549)

Note that ["RELAXNG-SCHEMA"](https://github.com/tobie/specref/blob/ebad7d8f7fdd1250451d69f32f3c87d9507914f1/refs/biblio.json#L2550-L2556) does reference iso.org.

Definitions can be updated via PR to the Specref repo.

**Comment by domel** [comment]
Yes, and I propose to change RELAXNG and RELAXNG-COMPACT to RELAXNG-SCHEMA (note that ISO/IEC 19757-2:2008 aka RELAXNG-SCHEMA defines RNG and RNC). The question is should I prepare a PR?

**Comment by domel** [comment]
@gkellogg Any thoughts?

**Comment by gkellogg** [comment]
RELAXNG-SCHEMA is the more up to date reference, and would make sense here and in RDF/XML.
